docs/features/UNIFIED_CANVAS.md

```diff
@@ -0,0 +1,96 @@
+The Unified Canvas is a tool designed to facilitate composition, enabling artists to use all available generation modes in a single unified workflow. This documentation will help explain the basics of using the Unified Canvas, as well as some advanced tools available to power users of the canvas.
+
+# Basics
+The Unified Canvas consists of two layers: The **Base Layer** and the **Mask Layer**, which can be toggled using the (Q) hotkey.
+
+### Base Layer
+The **Base Layer** is the image content currently managed by the canvas, and can be exported at any time to the gallery by using the **Save to Gallery** option. When the Base Layer is selected, the Brush (B) and Eraser (E) tools will directly manipulate the base layer. Any images uploaded to the canvas (or sent to the unified canvas from the gallery) will reset the canvas, and add the image as your initial base layer image.
+
+### Staging Area
+When you generate images, they will display in the canvas **Staging Area**, alongside the Staging Area toolbar buttons. While the Staging Area is active, you cannot interact with the canvas.
+
+Accepting generations will commit the new generation to the **Base Layer**. You can review all generated images using the Prev/Next arrows, save any individual generations to your gallery (without committing to the Base layer) or discard generations. While you can Undo a discard in an individual Canvas session, any generations that are not saved will be lost when the canvas resets.
+
+### Mask Layer
+The **Mask Layer** consists of any masks that have been created to inform Inpainting generations. You can generate a new mask by using the Brush tool with the Mask layer set as your Active layer. Masks will only be applied inside of the current bounding box.
+
+### Bounding Box
```

Comment:
Add brief explanation of what the user can accomplish by moving and resizing the bounding box.